File: 03.business-os/_archive/commonos_legacy_skeleton_20260425_184835/app_local/PocketSecretary/iphone/PocketSecretary/CommonOSTransport/CommonOSTransportRegistry.swift
import Foundation

enum CommonOSTransportRegistry {
    static func createDefault() -> CommonOSTransportDescriptor {
        CommonOSTransportDescriptor(
            appName: "PocketSecretary",
            requestIdField: "requestId",
            traceIdField: "traceId",
            syncStatusField: "syncStatus",
            clientTimestampField: "clientTimestamp",
            serverTimestampField: "serverTimestamp",
            itemsField: "items",
            nextPageTokenField: "nextPageToken",
            totalCountField: "totalCount",
            errorCodeField: "errorCode",
            errorMessageField: "errorMessage",
            errorDetailsField: "errorDetails"
        )
    }
}

File: 03.business-os/PocketSecretary/android/app/src/main/java/com/lsam/PocketSecretary/runtime/commonos/transport/CommonOSTransportRegistry.java
package com.lsam.PocketSecretary.runtime.commonos.transport;

public final class CommonOSTransportRegistry {

    public CommonOSTransportDescriptor createDefault() {
        return new CommonOSTransportDescriptor(
                "PocketSecretary",
                "requestId",
                "traceId",
                "syncStatus",
                "clientTimestamp",
                "serverTimestamp",
                "items",
                "nextPageToken",
                "totalCount",
                "errorCode",
                "errorMessage",
                "errorDetails"
        );
    }
}
